frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -446,7 +446,7 @@ export const Onboarding = () => {
               data-testid="subscription-item"
             >
               <Accordion.Control icon={<Icon name="subscription" />}>
-                {t`Get dashboard updates by email`}
+                {t`Get dashboard results by email or Slack`}
```

Comment:
"Results" is our internal terminology that I don't think matches the mental model of a new user - e.g. I have a dashboard, I want to get a fresh/updated dashboard every day. Dashboard is not something that produces results, rather an overview of the data?

Reply:
Interesting. I was thinking "updates" would mean when people _edited_ the dashboard. "Results" would refer to the data displayed on the dashboard. I'll change to "Subscribe to a dashboard by email or Slack."